employment-contract-no.md

```diff
@@ -0,0 +1,128 @@
+måneder i stillingen. Tid utover normal arbeidstid kompenseres ikke, verken i
+form av avspasering eller penger.
+
+Arbeidsgiver kan foreta trekk i lønn og feriepenger for alle beløp
+Arbeidstakeren er skyldig Bjerk, jfr. Arbeidsmiljølovens § 14-15 annet ledd.
+Bakgrunnen for dette er at Arbeidsgiver skal kunne trekke beløp for avtaler som
+er gjort med Arbeidstaker utover ordinær lønn.
+
+## Ferie- og godtgjørelse
+
+Ferie og feriegodtgjørelse reguleres etter bestemmelsene i ferieloven (29. april
+1988 nr 21), dog slik at ferie kun skal kunne tas ut når Bjerk AS’ virksomhet
+tillater dette. Ferieplaner skal koordineres mellom de ansatte, slik at alle
+nødvendige funksjoner til enhver tid er dekket.
+
+Årlig ferie utgjør som hovedregel 20 virkedager hvert ferieår.
+
+Bjerk holder stengt i fellesferier. Det vil si:
+
+- Sommerferie (15 feriedager)
+- Påskeferie (3 feriedager)
+- Juleferie (2-5 feriedager)
+
+Feriedager må taes ut iforbindelse med disse.
+
+Bjerk dekker bruk av mobil og -abonnement, internettabonnement og
+treningsabonnement. Konkrete abonnement og priser avtales med den enkelte
+ansatt.
+
+## Teknologioverførsel og -dokumentasjon
+
+Arbeidstakeren plikter i den utstrekning arbeidstakeren utvikler nye
+dataprogrammer, konsepter eller lignende, å utarbeide full dokumentasjon, så vel
+skriftlig som elektronisk (hvor relevant), for den teknologi og de
+forretningsideer som ligger til grunn for ny-utviklingen.
+
+Arbeidstakerens plikt i så måte er kollektiv og solidarisk med de øvrige
+ansatte.
+
+Dokumentasjonen skal redegjøre for Bjerk AS' virksomhet på en slik måte at den
+kan forstås og anvendes av personer med relevant bakgrunn/utdannelse. All slik
+nyutviklet programvare og dokumentasjon er Bjerk AS' eiendom, og kan ikke
+benyttes av Arbeidstakeren privat eller i annen virksomhet, under eller etter
+ansettelsen, uten at dette på forhånd er skriftlig godkjent av Bjerk AS'
+ledelse.
+
+## Lojalitets- og taushetsplikt
+
+Arbeidstakeren skal ikke i løpet av ansettelsestiden, på egen hånd eller sammen
+med andre, drive/bidra til noen form aktivitet som kan oppfattes som
+konkurrerende virksomhet. Arbeidstakeren forplikter seg til ikke å utøve slik
+aktivitet i inntil 6 måneder etter fratredelse dersom Bjerk aktiverer klausulen
+og betaler Arbeidstakeren 100 % lønn i karenstiden.
+
+Arbeidstakeren har så vel under som etter ansettelsesperioden taushetsplikt
+overfor alle utenforstående vedrørende Bjerk AS' teknologi, forretningsdrift og
+planer, og enhver annen opplysning som er viktig for Bjerk å beskytte i forhold
+til utenforstående. Denne bestemmelsen hindrer ikke Arbeidstakeren å gi
+opplysninger til offentlige myndigheter og domstoler, dersom arbeidstakeren
+etter lov eller annen regulering er forpliktet til dette.
+
+## Prøvetid og oppsigelse
+
+For stillingen gjelder seks måneders prøvetid med gjensidig oppsigelsesfrist på
+to uker. Etter prøvetid følger oppsigelsetid etter arbeidsmiljøloven.
```

Comment:
```suggestion
to uker. Etter prøvetiden er gjensidig oppsigelsestid tre måneder, regnet fra den
første i neste måned etter at oppsigelsen fant sted.
```

Jeg lurer litt på dette. Fordelen jeg ser med øke oppsigelsestiden er at det skaper ro rundt oppsigelser/bytte av arbeidssted. Allikevel, kan det kan kanskje avskrekke noen?

Er det en sjult risiko her? Jeg undres. Kanskje noen kan synse litt?

Reply:
Den vanlige rammen er fra 1-6 mnd oppsigelsestid. Hvor viktige stillinger for foretaket har lengst tid. Typisk kan være ansatte med særlig innsikt e.l
Den vanligste tiden er 3 mnd.
Sett mange ganger at den er 1 mnd første året og deretter økes til 3 mnd. 

Min konklusjon er at 3mnd etter prøvetid for utviklere er korrekt for begge parter

Reply:
For min egen del så føler jeg det er best med oppsigelsestid på tre måneder, igjen for å skape en trygghet/ro rundt det hele.

Reply:
@braaar har du noen tanker, eller skal vi lande denne?

Reply:
Tre måneder over fjølen høres enkelt og greit ut.